.github/workflows/centraldb_kind_test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build & Apply CentralDashboard manifests in KinD
+on:
+  pull_request:
+    paths:
+      - apps/centraldashboard/upstream/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install KinD
+      run: ./testing/gh-actions/install_kind.sh
+
+    - name: Create KinD Cluster
+      run: kind create cluster --config testing/gh-actions/kind-1-22.yaml
+
+    - name: Install kustomize
+      run: ./testing/gh-actions/install_kustomize.sh
+
+    - name: Install Istio
+      run: |
+        cd common/istio-1-11
+        kustomize build istio-crds/base | kubectl apply -f -
+        kustomize build istio-namespace/base | kubectl apply -f -
+        kustomize build istio-install/base | kubectl apply -f -
+
+    - name: Build & Apply manifests
+      run: |
+        cd apps/centraldashboard/upstream
+        kubectl create ns kubeflow
+        kustomize build overlays/kserve | kubectl apply -f -
```

Comment:
Let's have an extra check here with `kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout 180s` and ensure that all pods get created. We can set a timeout of 3 mins for now. This way we also ensure that the Pods can run as well

Reply:
I added this in all the gh actions.